docker/windows/Scripts/configure-tentacle.ps1

```diff
@@ -169,11 +126,68 @@ function Validate-Variables() {
   if($TargetName -ne $null) {
     Write-Log " - name '$env:TargetName'"
   }
+  if($TargetTenant -ne $null) {
+    Write-Log " - tenant '$env:TargetTenant'"
+  }
+  if($TargetTenantTag -ne $null) {
+    Write-Log " - tenant tag '$env:TargetTenantTag'"
+  }
+  if($TargetTenantedDeploymentParticipation -ne $null) {
+    Write-Log " - tenanted deployment participation '$env:TargetTenantedDeploymentParticipation'"
+  }
   if($null -ne $Space) {
     Write-Log " - space '$Space'"
   }
 }
 
+function Configure-Tentacle
```

Comment:
Moved methods in here around to match the Linux script which has the methods in the order they're called